startup/20-area-detectors.py

```diff
@@ -239,7 +239,71 @@ class EigerBaseV33(EigerBase):
     stats5 = Cpt(StatsPluginV33, 'Stats5:')
 
 
-class EigerSingleTrigger(SingleTrigger, EigerBaseV33):
+class EigerSingleTrigger(SingleTrigger, EigerBase):
```

Comment:
Do we still need this class?

Reply:
Right now they are both used. One for the Eiger 4M and one for the 1M:

eiger4m_single = EigerSingleTrigger_AD37('XF:11IDB-ES{Det:Eig4M}',                              
                                                            name='eiger4m_single')

eiger1m_single = EigerSingleTrigger('XF:11IDB-ES{Det:Eig1M}',
                                                             name='eiger1m_single')

In principle, I imagine that both detectors should use the same class but this needs to be tested (det operation as well as compatibility with the downstream analysis) at some point when we don't have users behind our shoulders.

Reply:
These classes need to stay in sync with the version of AD running on the IOC so if the IOCs are different versions then we need two classes on the python side.